docs/releases/node-releases.md

```diff
@@ -49,4 +50,126 @@ The following versions are no longer peered with by nodes running the active ver
+
+## Release Notes
+
+### V19.0
+
+#### Upgrade Notices
+
+**Version Limits**  
+Upgrades from versions V17.1 and to V19 will involve a sequential database upgrade and impact participation of the node on the network. RPC calls will be unavailable for a long period of time amongst other impacts.
+
+!!! warning "Upgrading from V17.1 and earlier to V19.0 not recommended"
+	It is highly recommended that nodes are upgraded to V18.0 first or a V18.0 ledger is acquired and used when upgrading to V19.0.
+
+**Confirmation tracking considerations**  
+The addition of confirmation height to the database requires the node to validate that blocks are confirmed before the cementing can occur. This process can take up to 24 hours or longer to complete but won’t impact participation on the network. For integrations watching confirmations, the existing [HTTP callback](/integration-guides/advanced/#http-callback), [block_confirm](/commands/rpc-protocol/#block_confirm) RPC and [confirmation_history](/commands/rpc-protocol/#confirmation_history) RPC methods will continue to function as before.
+
+!!! warning "Tracking confirmed block hashes required"
+	It is required that tracking of confirmed block hashes outside the node is done to avoid potential duplicate notifications from causing issues. This was a requirement in previous versions and remains the same with V19.
+
+For those looking to utilize the new WebSocket confirmation subscription or new confirmed option in [`block_info`](/commands/rpc-protocol/#block_info) RPC responses, special considerations should be taken if implementing before confirmation height updates are complete:
+
+* If the [websocket confirmation subscription](/integration-guides/advanced/#confirmations) is hooked up to receive all confirmations (default) then notifications for confirmations will come through during the cementing process on a new or upgrading ledger as the confirmation process will occur (it also fires for dependent confirmations)
+* Calls to [`block_info`](/commands/rpc-protocol/#block_info) for blocks in the ledger before the confirmation height upgrade process began may indicate `confirmed` as `false` despite their having been confirmed on the network before. This is expected behavior.
+* To validate that confirmation height upgrade is complete, note the `count` value from the [`block_count`](/commands/rpc-protocol/#block_count) RPC when the upgrade is started and once the `cemented` amount returned by this call (include the `include_cemented` option) is higher than that previous count, cementing is in sync.
+
+**Emitting nano_ prefixed addresses**  
+In this and future versions, all addresses emitted from the node will use the `nano_` prefix. It will continue to support input for `xrb_` prefixed addresses, but all services must verify they are properly set up to handle the node outputting `nano_` prefixed addresses.
+
+**Live network over TCP**  
```

Comment:
I wonder if this section should mention that serving bootstrap request can be disabled via `--disable_bootstrap_listener`), because some services may currently disable inbound TCP connections for this purpose (to reduce bandwidth use etc)

Reply:
I think we can save that callout for more permanent integration detail sections elsewhere.

Reply:
OK, good. I think it's important to have it somewhere because it says you can participate on the live network via TCP, yet avoid the bootstrap burden.